src/datePickerAndroid.rei

```diff
@@ -8,10 +8,7 @@ type action =
   | Dismissed
   | Set(response);
 
-type mode =
-  | Calendar
-  | Spinner
-  | Default;
+type mode = [ | `calendar | `spinner | `default];
```

Comment:
I'm not sure about this change. I don't think it's justified to change the public API in this case.

Reply:
@wokalski Sorry I didn't think about that. I am just using abstract and jsConverter everywhere. 

I just wanted to do the same I did for timePickerAndroid https://github.com/reasonml-community/bs-react-native/pull/233 If we are not using jsConverter here, should I change timePickerAndroid to use the same pattern as used here?

Reply:
Hmm, ok, that's a good point. Let's make this breaking change then!